server/templates/place.html

```diff
@@ -69,7 +69,7 @@ <h3 id="locale" data-lc="{{ locale }}"></h3>
       <div id="place-summary" class="row-col place-summary-container">{{ place_summary }}</div>
       <div id="main-pane" class="row"></div>
       {# TRANSLATORS: A message shown on the page while the content is loading. #}
-      <div id="page-loading" className="mt-4">{% trans %}Loading...{% endtrans %}</div>
+      <div id="page-loading" className="mt-4">{% trans %}<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 -960 960 960" width="24px" fill="#5f6368"><path d="M480-80q-82 0-155-31.5t-127.5-86Q143-252 111.5-325T80-480q0-83 31.5-155.5t86-127Q252-817 325-848.5T480-880q17 0 28.5 11.5T520-840q0 17-11.5 28.5T480-800q-133 0-226.5 93.5T160-480q0 133 93.5 226.5T480-160q133 0 226.5-93.5T800-480q0-17 11.5-28.5T840-520q17 0 28.5 11.5T880-480q0 82-31.5 155t-86 127.5q-54.5 54.5-127 86T480-80Z"/></svg><p>Loading</p>{% endtrans %}</div>
```

Comment:
Just wrap the {% trans %}  tag around the actual text, in this case the word "Loading"

Reply:
Can you add comment describing the SVG and give a source reference where it came from?